wire-ios/Wire-iOS Tests/UserClientCellTests.swift

```diff
@@ -20,64 +20,125 @@ import XCTest
 @testable import Wire
 
 final class UserClientCellTests: BaseSnapshotTestCase {
+    var sut: UserClientCell!
+    var container: UIView!
 
-    func cell(_ configuration: (UserClientCell) -> Void) -> UserClientCell {
-        let cell = UserClientCell(frame: CGRect(x: 0, y: 0, width: 320, height: 64))
-        configuration(cell)
-        cell.layoutIfNeeded()
-        return cell
+    override func setUp() {
+        super.setUp()
+        let frame = CGRect(x: 0, y: 0, width: 390, height: 84)
+        sut = UserClientCell(frame: frame)
+        container = containerView(with: sut, snapshotBackgroundColor: nil)
+        container.frame = frame
     }
 
```

Comment:
Since we do have a setUp method it would be a good idea to have a tearDown as well and set our properties to nil.

Reply:
done